org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -423,6 +431,22 @@ private synchronized void doNodeInitialisation() {
         coordinatorHandler.serializeNetwork();
     }
 
+    private synchronized void reinitializeDevice() {
```

Comment:
I do not completely understand the workings of this method. It reinitializes the complete node (via `initialiseZigBeeNode`), and adds some listeners, but does not uninitialise everything before (e.g., the channel converters are not disposed). Why is this OK?

I had thought that one could either dispose all channel converters and them reinitialize them (including device initialization), or that one could dispose the complete thing handler and reinitialize ist.

WDYT?

Reply:
You're right, something went totally wrong here. Will change it to this:
```
    private synchronized void reinitializeDevice() {
        dispose();
        initialize();
    }
```